app/controllers/admin/ambassador_task_assignments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Admin::AmbassadorTaskAssignmentsController < Admin::BaseController
+  include SortableTable
+  layout "new_admin"
+
+  def index
+    @ambassador_task_assignments =
+      Kaminari
+        .paginate_array(sorted_assignments)
+        .page(params.fetch(:page, 1))
+        .per(params.fetch(:per_page, 25))
+  end
+
+  private
+
+  def sortable_columns
+    %w[completed_at organization_name task_title ambassador_name]
+  end
+
+  def sorted_assignments
+    assignments =
+      AmbassadorTaskAssignment
+        .includes(:ambassador, :ambassador_task)
+        .completed
+
+    sort_criterion = params.fetch(:sort, "").to_sym
+    direction = params.fetch(:direction, "").to_sym
```

Comment:
There are already helper methods (in `SortableTable`) for these things, `sort_column` and `sort_direction`

Reply:
Which do two important things:

1. Make sure we only execute the whitelisted list of sequel sorts
2. provide a default